examples/orbit_5k_pipeline.ipynb

```diff
@@ -4,18 +4,32 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Orbit5K analysis (WIP)\n",
+    "# Tutorial : Classifying Orbit5k with giotto-deep\n",
     "\n",
-    "#### Author: Raphael Reinauer\n",
+    "#### Authors : Nicolas Berkouk and Raphael Reinauer\n",
     "\n",
-    "In this notebook we provide the code for the `orbit_5k` analysis using Persformer.\n",
     "\n",
-    "The`orbit_5k` dataset consists of simulation of the dynamical system:\n",
-    "$$ y = \\alpha x mod 1, y = x mod 1$$\n",
+    "The Orbit5k dataset consists 5000 subsets of $[0,1]^2$, each containing $1000$ points. For a given $\\rho \\in \\{2.5, 3.5, 4.0, 4.1,4.3\\}$, each subset $S^\\rho \\subset [0,1]^2 $ is generated randomly according to the following procedure:\n",
     "\n",
-    "Depending on the value of the parameter $\\alpha$, the orbits hahve a different dynamics.\n",
+    "1. Start with a uniformly randomly sampled point $(x_0,y_0)\\in [0,1]^2$\n",
+    "2. For $n \\geq 1$, generate $(x_{n+1},y_{n+1})$ inductively by : \n",
     "\n",
-    "The goal of ths notebook is to classify the orbits and infer the parameter $\\alpha$ that was used to generate such orbits."
+    "$$ x_{n+1} = x_{n} + \\rho y_n(1-y_n) ~~~\\text{mod }1$$\n",
+    "$$ y_{n+1} = y_{n} + \\rho x_n(1-x_{n+1}) ~~~\\text{mod }1$$\n",
+    "\n",
+    "3. Define $S^\\rho := \\{(x_i,y_i) \\mid i = 0...999 \\}$\n",
```

Comment:
In this version it's a 1.
https://github.com/giotto-ai/giotto-deep/blob/notebook_orbit5k/examples/orbit_5k_pipeline.ipynb
![image](https://user-images.githubusercontent.com/16176161/178743921-d2773705-91c0-4abd-9a83-589c78c50d8c.png)